uvicorn/protocols/http/h11_impl.py

```diff
@@ -417,12 +454,16 @@ async def send(self, message):
             if message_type != "http.response.start":
                 msg = "Expected ASGI message 'http.response.start', but got '%s'."
                 raise RuntimeError(msg % message_type)
+            message = cast(HTTPResponseStartEvent, message)
```

Comment:
here we can cast because we're sure that is is a HTTPResponseStartEvent ?

Reply:
Exactly! I guess at some point, mypy will be able to infer that based on the conditional...